gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/RetrofitConfig.groovy

```diff
@@ -18,7 +18,12 @@ package com.netflix.spinnaker.gate.config
 
 import com.netflix.spectator.api.Registry
 import com.netflix.spinnaker.config.OkHttpClientConfiguration
+import com.squareup.okhttp.Interceptor
+import com.squareup.okhttp.MediaType
 import com.squareup.okhttp.OkHttpClient
+import com.squareup.okhttp.Request
+import com.squareup.okhttp.RequestBody
+import okio.Buffer
```

Comment:
Would recommend just reverting this class.

Reply:
Yep, sorry. Rushing never helps... 🙁